auth.go

```diff
@@ -8,13 +8,23 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"strings"
 	"time"
 
 	"github.com/nlopes/slack"
 
 	log15 "gopkg.in/inconshreveable/log15.v2"
 )
 
+const (
```

Comment:
Talked in person, disregard this as `strings.Join` will not be applied and it will be a mess